src/IndexManagement/IndexManager.php

```diff
@@ -236,6 +236,32 @@ function () use ($source, $destination) {
         );
     }
 
+    public function putSettings(string $index, array $settings = []): void
+    {
+        $body = ['index' => []];
+
+        foreach ($settings as $key => $setting) {
```

Comment:
is the goal of this to allow only `refresh_interval` and `number_of_replicas` as settings?

Reply:
Once the User Story requirement is to change only these properties I feel safer to do not allow clients to put other settings than those through this package.